### DISCUSSION TITLE
Can't login after using docker-compose up with admin_password specified in docker-compose.yaml

### DISCUSSION THREAD

**Discussion by MaxwellPSanders** [mediacms-io/mediacms#465] [Q&A] [answered]
I am very new to Docker, so I am sure I am just missing something.
I followed the instructions of the docker installation, but after I did **docker-compose up** I realized I needed to make a port change, so I booted it back down, and forgot to take note of what the admin password was from the migrations.
I do not know what the admin password is so I uncommented the admin_password from the **docker-compose.yaml** file and ran **docker-compose up** and I am still not able to log in to my instance.

Thank you for any help!

**Comment by mgogoulos** [comment, marked as answer]
Try resetting the password, using instructions on https://github.com/mediacms-io/mediacms/discussions/81
you have to bash to the web container and run the command through Django shell

**Reply by MaxwellPSanders** [reply]
Thank you, I had to figure out how to bash into the web container, but once I did everything worked great!
